WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterViewModel.kt

```diff
@@ -156,13 +159,26 @@ class ActivityLogTypeFilterViewModel @Inject constructor(
             val loadingText: UiString = UiStringRes(R.string.loading)
         }
 
-        data class Error(val retryAction: Action) : UiState() {
+        sealed class Error : UiState() {
             override val errorVisibility = true
+            abstract val image: Int
```

Comment:
Small nit-pick - could this be marked as "@DrawableRes" ?

Reply:
AFAIK since it's an abstract field it can't be annotated with @DrawableRes :(. Java doesn't know abstract fields so they are not implemented as actual fields -> the @DrawableRes annotation can be used only on `METHOD, PARAMETER, FIELD, LOCAL_VARIABLE`. 
I added the annotation at least to the subclasses in https://github.com/wordpress-mobile/WordPress-Android/pull/13707/commits/931c81b2110adf77d81663b1f88efdf508b65dc6. I could also make the abstract field open and assign `0` as default value. The annotation would work in that case. Let me know if you prefer that.

Reply:
> Java doesn't know abstract fields so they are not implemented as actual fields -> the @DrawableRes annotation can be used only on `METHOD, PARAMETER, FIELD, LOCAL_VARIABLE`.

 <shakes fist at Java> - You are correct. 

> I added the annotation at least to the subclasses
Makes sense here. Thanks 👍